staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types_jsonschema.go

```diff
@@ -59,7 +59,7 @@ type JSONSchemaProps struct {
 // JSON represents any valid JSON value.
 // These types are supported: bool, int64, float64, string, []interface{}, map[string]interface{} and nil.
 type JSON struct {
-	Raw []byte `protobuf:"bytes,1,opt,name=raw"`
+	Raw []byte `json:"raw,omitempty" protobuf:"bytes,1,opt,name=raw"`
```

Comment:
there is existing data serialized this way, and we *cannot* break that. either leave the tags off, or add them matching the current schema and stored data. we can and should fix this in v1, and should add a linter immediately to prevent any more fields like this from occurring (I really thought we had one already)

Reply:
I would add the tag, even with the wrong cases. And add a TODO here as well. Finally, we need a bold release-note telling people that they have to rewrite their CRD validations.

Dirty hack alternative: add both case-variants as fields and fix on conversion. @liggitt how bad do you feel about that?

Reply:
In other words, with the hack: we either loose data on the etcd side or on the user side with his manifests. Choose one!

Reply:
@roycaihw Would the linter you are working on have caught this problem?